### PR TITLE
Isolate Machine with Cybereason from Slack

### PR DESCRIPTION
**Pull request by mrinehart-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Isolate_Machine_with_Cybereason_from_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
+* Cybereason account
+
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, you must configure the Workflow Parameters for the Slack chatops command as well as the channel the chatops bot should monitor.
```
Comment (pmara-r7) [inline workflows/Isolate_Machine_with_Cybereason_from_Slack/help.md:24]:
```suggestion
Once the workflow has been imported, you must configure the Workflow Parameters for the Slack chatops command and the channel the chatops bot should monitor. 
```

Comment (pmara-r7) [reply workflows/Isolate_Machine_with_Cybereason_from_Slack/help.md:24]: sentence reads a little weird. I think this helps

**Comment by pmara-r7** [comment]
One minor wording thing, other than that lgtm

**Comment by mrinehart-r7** [comment]
> One minor wording thing, other than that lgtm

Good comment, committed!

**Review by pmara-r7 — APPROVED** [review]
LGTM

**Review by mberezin-r7 — APPROVED** [review]

**Pull request merged**